package.json

```diff
@@ -10,6 +10,7 @@
     }
   ],
   "devDependencies": {
+    "axios": "^0.15.3",
```

Comment:
Like I said, for production it should be included in the build process no? At least I think that's what webpack is going to do. This will be a static app.